README.md

```diff
@@ -2,7 +2,7 @@
 
 
 <p align="center" width="100%">
-    <img width="300" src="https://github.com/the-browser-toolbox/to-do/blob/main/icon.jpg"> 
```

Comment:
This doesn't look right.

Reply:
what doesn't look right

Reply:
Has to be a correct way. Using the image in our Readme from your fork doesn't make sense to me at all. 

Maybe just upload it elsewhere and use the external link?

If your fork is deleted the image shouldn't work for the other repo, should it?

Reply:
@maximilianschmelzer let @adityabisoi take a look at this as he is the owner of the repo.

Reply:
and its better if you take care of the issue you have taken #23 as it's not working

Reply:
Interesting take on open source collab. But ok, I know when it has no point in continuing to explain 🙃

@adityabisoi Please read my explanation and write your own I guess. Then it can be worked on... 🤣